src/components/LanguageMenu/LanguageMenu.tsx

```diff
@@ -85,20 +88,31 @@ export class LanguageMenu extends Component<LanguageMenuProps> {
       logger.warn(`Menu '${name}' does not contain items`);
       return null;
     }
-
     return (
       <HtmlSpan className={classnames(className, baseClassName)}>
         <Menu>
-          <MenuButton {...passProps} className={menuButtonClassName}>
-            {name}
-          </MenuButton>
-          {!!MenuListComponentReplace ? (
-            <MenuListComponentReplace {...menuListProps}>
-              {children}
-            </MenuListComponentReplace>
-          ) : (
-            <MenuList {...menuListProps}>{children}</MenuList>
-          )}
+          {({ isOpen }: { isOpen: boolean }) => {
+            return (
+              <Fragment>
+                <MenuButton
+                  {...passProps}
+                  className={classnames(
+                    menuButtonClassName,
+                    isOpen ? menuButtonOpenClassName : '',
```

Comment:
Not a real problem but possible alternative solution to prevent "empty" ternary operation choice is by changing to:
`isOpen && menuButtonOpenClassName,`